component/org.wso2.carbon.identity.dpop/src/main/java/org/wso2/carbon/identity/dpop/token/binder/DPoPBasedTokenBinder.java

```diff
@@ -256,4 +264,26 @@ public String[] getAllGrantTypes() {
         }
         return supportedGrantTypesSet.toArray(new String[supportedGrantTypesSet.size()]);
     }
+
+    private static boolean skipDPoPValidationInRevoke() {
+
+        Object skipDPoPValidationInRevokeObject = IdentityUtil.readEventListenerProperty
+                        (AbstractIdentityHandler.class.getName(), OauthDPoPInterceptorHandlerProxy.class.getName())
+                .getProperties().get(DPoPConstants.SKIP_DPOP_VALIDATION_IN_REVOKE);
+
+        if (skipDPoPValidationInRevokeObject == null){
+            return DPoPConstants.DEFAULT_SKIP_DPOP_VALIDATION_IN_REVOKE_VALUE;
+        }
+
+        String skipDPoPValidationInRevokeValue = skipDPoPValidationInRevokeObject.toString().trim();
+
+        if (!(skipDPoPValidationInRevokeValue.equals("true") || skipDPoPValidationInRevokeValue.equals("false"))) {
```

Comment:
to avoid a NPE lets change this other way around.
"true" -> equals(skipDPoPValidationInRevokeValue)

Reply:
Fixed